### ISSUE TITLE
[BITV] 9.1.4.3/1.1 - The error message "Wrong username or password" does not meet the contrast requirements for text - it is 3,7:1. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41848]
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-6fe46b78da-v1-n1

![image](https://github.com/nextcloud/server/assets/6078378/7a078866-99e6-4d83-ad01-de4e9dd32a0d)
